### PR TITLE
Add business verification

### PR DESCRIPTION
**Pull request by sadick254**
## Summary.
Add support for business verification jobs.

### CODE REVIEW

File: src/business-verification.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      path,
+      method: 'POST',
+      headers: {
+        'Content-Type': "application/json"
+      }
+    };
+
+    return new Promise((resolve, reject) => {
+      let json = '';
+      const req = https.request(options, function(resp) {
```
Comment (mldangelo) [inline src/business-verification.js:24]: Can you use axios here?

File: src/business-verification.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const https = require('https');
+const JOBTYPE = require('./constants/job-type');
+
+class BusinessVerification {
+  constructor() {
+    this.id_types = ['BASIC_BUSINESS_REGISTRATION', 'BUSINESS_REGISTRATION','TAX_INFORMATION'];
+    this.business_types = ['co', 'bn', 'it'];
```
Comment (mldangelo) [inline src/business-verification.js:7]: please don't hardcode these

File: src/constants/job-type.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const JOBTYPE = {
```
Comment (mldangelo) [inline src/constants/job-type.js:1]: See https://github.com/smileidentity/smile-identity-core-js/blob/main/src/constants.ts

**Review by mldangelo — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts and re-write your changes in typescript.

**Comment by sadick254** [comment]
Closing in favour of #85

**Pull request closed without merging**